### PR TITLE
GC unused typeddefs and params.

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR

### CODE REVIEW

File: searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h [hunk 1]
```diff
@@ -113,8 +111,8 @@ class PostingListSearchContextT : public PostingListSearchContext
     static const long MIN_APPROXHITS_TO_NUMDOCS_RATIO_BEFORE_APPROXIMATION = 10;
 
     PostingListSearchContextT(const Dictionary &dictionary, uint32_t docIdLimit, uint64_t numValues,
-                              bool hasWeight, const PostingList &postingList, const IEnumStore &esb,
-                              uint32_t minBvCocFreq, bool useBitVector, const ISearchContext &baseSearchCtx);
+                              bool hasWeight, const PostingList &postingList, uint32_t minBvCocFreq,
```
Comment (toregge) [inline searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h:114]: No need to keep old misspellings, consider using `minBvDocFreq`

File: searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h [hunk 2]
```diff
@@ -151,8 +149,8 @@ class PostingListFoldedSearchContextT : public PostingListSearchContextT<DataT>
     using Parent::singleHits;
 
     PostingListFoldedSearchContextT(const Dictionary &dictionary, uint32_t docIdLimit, uint64_t numValues,
-                                    bool hasWeight, const PostingList &postingList, const IEnumStore &esb,
-                                    uint32_t minBvCocFreq, bool useBitVector, const ISearchContext &baseSearchCtx);
+                                    bool hasWeight, const PostingList &postingList, uint32_t minBvCocFreq,
+                                    bool useBitVector, const ISearchContext &baseSearchCtx);
```
Comment (toregge) [inline searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h:153]: No need to keep old typos, consider using `minBvDocFreq`

**Review by toregge — APPROVED** [review]

File: searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h (hunk 1 above)
Comment (baldersheim) [reply searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h:114]: Fixed

File: searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h (hunk 2 above)
Comment (baldersheim) [reply searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h:153]: Fixed

**Pull request merged**